gnosis/safe-react
# v1.3.0 smart contract compatibility

## What it solves
Closes #2216 adding compatibility for v1.3.0 smart contracts

## How this PR fixes it
It updates off chain signature parameters, that will need chainId from v1.3.0 but not on previous versions
It also adds different ABI loading depending on current safe version, so we ensure that we use the correct ABI for each case.

## How to test it
Create a new account on Rinkeby and validate that it's still generated using v1.1.1
Ensure that transaction signing and off chain signing is working as expected.
A v1.3.0 should be created manually or ask @dasanra for it.
Ensure that transaction signing and off chain signing is working as expected.

## Any extra information
